### ISSUE TITLE
Userstory 7: Image import

### ISSUE THREAD

**Issue by IrisHerholz** [IrisHerholz/capstone-project#13]
## Value Proposition

As a User
I want to to be able to upload images to my entries
so that i can always have a look at my past entries

## Description

![Bildschirmfoto 2023-02-06 um 11 23 09](https://user-images.githubusercontent.com/118169582/216947240-0d5eacf9-ee59-43ce-bea1-eecb0c777ed4.png)

## Acceptance Criteria

- an input field is displayed at the submitted entry form on the entry page 
- a cancel button is added on the form page
- if you click on the image it opens a modal with your image in full size
- on the modal is a delete and edit button shown

## Tasks

- implement local storage
-  install and import cloudinary and formidable into this file
-  outside of the pages folder create a env.local
- write the keys from cloudinary in the env.local
-  create a api folder in the pages folder with a upload.js file
-  write a handler function  in the upload.js for uploading the files to cloudinary
-  inside of the Modal:
 create a close button
 create a edit button 
 create a delete button and handle the onClick so that it deletes the image
- write for all features handle functions 

## Complexity

XL

**Comment by Long-HaHoang** [comment]
Hi Iris,
I think your user story is a bit too big for one.
You might split the picture uploads and the modal picture view. 
This can expedite your code review later on. Less is often more. 
It's will be easier for us if we can just look at one feature for example only the picture upload.
But it depends on you, and how confident you are with modals in the last user story.

**Comment by IrisHerholz** [comment]
Ok, i will see how everything works after finishing US 6. And maybe I'll adjust it then! Thank you :)

**Comment by Long-HaHoang** [comment]
> so that i can always have a look at my past entries

I don't understand this part and there is a typo `i` instead of `I`
What's the actual goal?

You can rephrase it to make it more precise since I can already see past entries.
> so I can differentiate between past entries better
